.github/workflows/basic.yml

```diff
@@ -103,7 +103,7 @@ jobs:
       - name: Run tests
         working-directory: ${{env.BUILD_DIR}}
         run: |
-          ctest --output-on-failure
+          ctest --output-on-failure --verbose
```

Comment:
Why `--verbose` ? `--output-on-failure` is enough - it prints the verbose output when a test fails.